src/index.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom';
 
 import App from 'components/App';
-import './index.scss';
+import store from 'redux/store';
+import 'index.scss';
 
 if (typeof window !== 'undefined') {
   global.document.addEventListener('DOMContentLoaded', () => {
     const mountNode = global.document.getElementById('app');
 
-    ReactDOM.render(<App />, mountNode);
+    ReactDOM.render(<Provider store={store}><App /></Provider>, mountNode);
```

Comment:
We wouldn't want to do this if we weren't using Redux. This makes the store available to connected components. More info here: https://github.com/reactjs/react-redux/blob/master/docs/api.md#provider-store